### ISSUE TITLE
Remove Message.ProductUrl from Product.AskQuestion.xml and use Product.Url instead

### ISSUE THREAD

**Issue by Michael-Herzog** [smartstore/Smartstore#567]
Write migration to fix this value in the database.

Once done revert this commit:
https://github.com/smartstore/Smartstore/commit/70297c07001a3262f3ee886683340d2dbdbf6f00

**Comment by mgesing** [comment]
Probably solved by 99c46be.

**Comment by Michael-Herzog** [comment]
No. The param is still used in the template Product.AskQuestion.xml. The goal of this issue was to get rid of it.
